### PR TITLE
Support For Bulk API Endpoints

### PR DESCRIPTION
**Pull request by sephiroh**
Support for the Bulk API Endpoints from TestRail Release 7.2.1

### CODE REVIEW

File: TestRail/TestRailClient.cs [hunk 1]
```diff
@@ -1113,6 +1115,48 @@ private RequestResult<T> _SendGetCommand<T>(string uri)
...
+        /// <param name="uri">The endpoint to send the request to.</param>
+        /// <param name="type">The type of the JSON Bulk Property to look for (e.g. cases, projects, etc.)</param>
+        /// <returns>The List result of the request.</returns>
+        private RequestResult<T> _SendGetCommand<T>(string uri, CommandAction type)
+        {
+            var strType = type.ToString().ToLower();
+            var temp = _SendGetCommand<BulkAPI>(uri);
+            var list = WrappedList<T>(temp.Payload.DataItems[strType].ToString());
+
+            while (temp.Payload._Links.Next != null)
```
Comment (Brrovko) [inline TestRail/TestRailClient.cs:1129]: Is there a limit for API to return values?

Comment (sephiroh) [reply TestRail/TestRailClient.cs:1129]: TestRail 7.2.1 Release automatically limits the bulk API Endpoints to 250 records, and but will include the next 250 URI Link on the response payload. The code will loop and check until the "Next" property is not null (meaning until there is no more records to be retrieved). Did I answer your question?

Comment (Brrovko) [reply TestRail/TestRailClient.cs:1129]: Yes, thanks. 
I used your code in my repo because looks like the project are abandoned.

**Review by Brrovko — APPROVED** [review]

**Comment by Brrovko** [comment]
@Z3L1A  @Astro03 Can you merge the PR?

**Comment by sephiroh** [comment]
#65 Potentially closing this issue

**Pull request merged**